jetty-ee10/jetty-ee10-websocket/jetty-ee10-websocket-jetty-client/pom.xml

```diff
@@ -42,7 +42,6 @@
     <dependency>
       <groupId>org.eclipse.jetty.ee10</groupId>
```

Comment:
This module has nothing else except the `JettyWebSocketConfiguration` class. So you can't split this out or this would be renamed to `jetty-ee10-jetty-client-webapp` and `jetty-ee10-jetty-client` would be completely empty.

Reply:
Please rename this module to `*-webapp`, so it is symmetric with ee9.